### ISSUE TITLE
FRASER bam files for samples

### ISSUE THREAD

**Issue by jjyotikataria** [gagneurlab/FRASER#8]
It will be really helpful if you can provide the bam files for all the example data which includes sample4,sample5....sample9.

**Comment by ischeller** [comment]
Hi @jjyotikataria,

Can you describe why that would be helpful for you? Originally we included all of those example bam files in this package, but we had to reduce the number of example bam files that we provide in order to meet the bioconductor package guidelines.

**Comment by jjyotikataria** [comment]
@ischeller I wanted to confirm some events by visualizing the results in Igv. Is there any way of sharing those bam files as of now?

**Comment by c-mertes** [comment]
dear @jjyotikataria,
the bamfiles we provide within the package are only there for demonstration of the functionality of the package and should not be used to investigate outliers. If you want to check real outliers on real data, I would suggest to run it on the GEUVADIS dataset. The bam files are public and hence the results can be fully analyzed within IGV. 

https://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-6/files/processed/

To see how to run it on a subset have a look on our colab tutorial (http://tinyurl.com/RNA-ASHG-colab). 
Let us know if you need assistance with the geuvadis dataset. If you still want to have a look at the bam files you can just look at the bam files within the package. keep in mind they are just a fraction of the real one, which can not be shared unfortunately.

**Comment by jjyotikataria** [comment]
dear @c-mertes 

Thank you for the elaborate reply. My goal was not to detect the outliers but to just visualize some specific events from the detected event lists and have a visualization to interpret graphically via Sashimi plots in IGV. But thats fine, if the other bam files cannot be shared. I have assessed the bam files for sample1,2 and 3 in the package already.

I would love to see the GEUVADIS data set results. I went through the collab tutorial. Do I need to setup the IR notebook?
